### PR TITLE
Fix glossary references in `redirect.yaml`

### PR DESCRIPTION
**Pull request by DoryanR**
part of #4804

### CODE REVIEW

File: wiki/redirect.yaml [hunk 1]
```diff
@@ -228,155 +226,147 @@
...
+"test_mode":                   "Beatmap_Editor/Test_mode"
+"time_signature":              "Beatmapping/Time_signature"
+"timing_section":              "Beatmapping/Timing_section"
+"touchpad":                    "Gameplay/Input_device/Graphics_tablet"
+"touchscreen":                 "Gameplay/Input_device/Graphics_tablet"
+"touhosu!":                    "Glossary/Touhosu!"
 "wip":                         "Beatmap/Category#work-in-progress-and-pending"
 "work_in_progress":            "Beatmap/Category#work-in-progress-and-pending"
-"z/x":                         "Glossary#z/x"
+"z/x":                         "Play_style"
```
Comment (TicClick) [inline wiki/redirect.yaml:369]: I guess that's a good alternative

**Review by TicClick — CHANGES_REQUESTED** [review]

**Comment by TPGPL** [comment]
The redirects with `_glossary`/`glossary_` in name should be removed i guess

**Review by TicClick — DISMISSED** [review]
the removed redirects, such as AFK, are not osu!-specific, or are too specific (like `#looping`, which links to a 30s-themed cut from the ranking criteria).

on top of that, my wild guess is that we don't need any `*_glossary` (or `glossary_*`) redirects either, as they duplicate links to existing articles and were unlikely made for meaningful usage.

need someone else to give it a read, but here's an approval from me

File: wiki/redirect.yaml [hunk 2]
```diff
@@ -213,7 +213,7 @@
 "xk":                          "Game_modifier/xk"
 
 "acc":                         "Accuracy"
-"accuracy_glossary":           "Accuracy"
+"accuracy":                    "Accuracy"
```
Comment (TicClick) [inline wiki/redirect.yaml:216]: this is unnecessary, osu! wiki picks up any case

**Review by TicClick — CHANGES_REQUESTED** [review]

File: wiki/redirect.yaml (hunk 2 above)
Comment (DoryanR) [reply wiki/redirect.yaml:216]: I had forgotten to remove it, as for `nc`

**Review by TPGPL — DISMISSED** [review]
lgtm

**Review by TPGPL — APPROVED** [review]

**Pull request merged**